content/en/examples/service/access/hello-application.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: hello-world
-          image: gcr.io/google-samples/node-hello:1.0
+          image: us-docker.pkg.dev/google-samples/containers/gke/hello-app:2.0
```

Comment:
How can we confirm the new image is correct? Does the [tutorial](https://kubernetes.io/docs/tasks/access-application-cluster/service-access-application-cluster/) still run as expected with this updated image, without any changes to the instructions or output?